docs/13-walkthroughs/3-add-logging.md

```diff
@@ -44,12 +64,50 @@ To open Grafana you can use the `zarf connect logging` command.
 
 You'll be redirected the `/login` page where you have to sign in with the Grafana credentials you saved [in a previous step](#note-the-credentials).
 
+![zarf operator logging into Loki](../.images/walkthroughs/logging_login.png)
+
 Once you've successfully logged in go to:
 
 1. The "Explore" page (Button on the left that looks like a compass)
 
-1. you can select `Loki` in the dropdown, and then
+2. you can select `Loki` in the dropdown, and then
+
+3. enter `{app="game"}` into the Log Browser query input field
 
-1. enter `{app="game"}` into the Log Browser query input field
+![zarf operator typing {app='game'} into the Loki Log Browser](../.images/walkthroughs/logging_query.png)
 
 Submit that query and you'll get back a dump of all the game pod logs that Loki has collected.
+
+![Loki query results](../.images/walkthroughs/logging_logs.png)
+
+## Removal
+
+1. Use the `zarf package list` command to get a list of the installed packages and their compoents.  This will give you the name of the init package and the logging component to remove it.
+
+<iframe src="/docs/walkthroughs/logging_list.html" width="100%"></iframe>
+
+2. Use the `zarf package remove` command to remove the `logging` component from the init package.  Don't forget the `--confirm` flag.  Otherwise you'll receive an error.
+
+<iframe src="/docs/walkthroughs/logging_remove.html" width="100%" height="120px"></iframe>
+
+
+## Troubleshooting
+
+### Unable to connect to the Kubernetes cluster, or timed out waiting for cluster to report healthy.
```

Comment:
Same thing about hanging chad wrapping here
![image](https://user-images.githubusercontent.com/3977569/227033347-33c0d77e-a2a8-485a-9b58-b179ece62b13.png)

Reply:
Made this shorter